client/me/purchases/concierge-banner/index.jsx

```diff
@@ -3,39 +3,149 @@
  */
 import React, { Component } from 'react';
 import { localize } from 'i18n-calypso';
+import classnames from 'classnames';
 
 /**
  * Internal Dependencies
  */
+import Card from 'components/card';
 import ActionCard from 'components/action-card';
 import TrackComponentView from 'lib/analytics/track-component-view';
+import conciergeImage from 'assets/images/illustrations/jetpack-concierge.svg';
+import {
+	CONCIERGE_HAS_UPCOMING_APPOINTMENT,
+	CONCIERGE_HAS_AVAILABLE_INCLUDED_SESSION,
+	CONCIERGE_HAS_AVAILABLE_PURCHASED_SESSION,
+	CONCIERGE_SUGGEST_PURCHASE_CONCIERGE,
+} from 'me/concierge/constants';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
 
 class ConciergeBanner extends Component {
+	placeholder() {
+		return (
+			<Card compact={ false }>
+				<div className="concierge-banner__placeholders is-placeholder">
+					<div className="concierge-banner__placeholder-row-container is-placeholder">
+						<div className="concierge-banner__placeholder-row is-placeholder" />
+						<div className="concierge-banner__placeholder-row is-placeholder" />
+						<div className="concierge-banner__placeholder-row is-placeholder" />
+					</div>
+					<div className="concierge-banner__placeholder-button-container">
+						<div className="concierge-banner__placeholder-button is-placeholder" />
+					</div>
+				</div>
+			</Card>
+		);
+	}
+
+	getBannerContent() {
+		const { bannerType, translate } = this.props;
+
+		let headerText, mainText, buttonText, buttonHref, illustrationUrl;
+
+		switch ( bannerType ) {
+			case CONCIERGE_HAS_UPCOMING_APPOINTMENT:
+				headerText = translate( 'Your appointment is coming up!' );
+				mainText = translate(
+					'Get ready with your questions for your upcoming Quick Start session appointment.',
+					{
+						comment:
+							"Please extend the translation so that it's clear that these sessions are only available in English.",
+					}
+				);
+				buttonText = translate( 'Session dashboard' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'View your session dashboard' )` *ES Score: 7*